src/components/OSCALCatalog.js

```diff
@@ -17,6 +17,10 @@ const useStyles = makeStyles((theme) => ({
 export default function OSCALCatalog(props) {
   const classes = useStyles();
 
+  useEffect(() => {
+    if (props.setContentLoaded) props.setContentLoaded(true);
+  }, []);
+
```

Comment:
```suggestion
  useEffect(() => {
    if (props.setContentLoaded) { 
       props.setContentLoaded(true);
    }
  }, []);
```
Kinda small, but I think that the braces add a lot to readability.